lib/plugins/aws/lib/monitorStack.js

```diff
@@ -80,9 +88,34 @@ module.exports = {
                   let eventLog = `CloudFormation - ${eventStatus} - `;
                   eventLog += `${event.ResourceType} - `;
                   eventLog += `${event.LogicalResourceId}`;
-                  this.serverless.cli.consoleLog(eventLog);
+                  legacy.consoleLog(eventLog);
                 } else {
-                  this.serverless.cli.printDot();
+                  legacy.consoleLog(chalk.yellow('.'));
+                }
+
+                if (event.ResourceType !== 'AWS::CloudFormation::Stack') {
+                  if (eventStatus && eventStatus.endsWith('PROGRESS')) {
+                    inProgressResources.add(event.LogicalResourceId);
+                  } else if (eventStatus && eventStatus.endsWith('COMPLETE')) {
+                    completedResources.add(event.LogicalResourceId);
+                    inProgressResources.delete(event.LogicalResourceId);
+                  }
+                }
+
+                const operationPhrase = (() => {
+                  if (action === 'create') return 'Creating';
+                  if (action === 'update') return 'Updating';
+                  return '';
+                })();
+
+                if (operationPhrase && (completedResources.size || inProgressResources.size)) {
```

Comment:
If I see correctly we never use literally `operationPhrase`, and the only place where it's used this condition.

In such case, I think it'll be cleaner to simply directly check `action === 'create' || action === 'update'`, it'll be less confusing (I was wondering why we need this capitalized string)

Reply:
that's an ommision on my part - we should use this phrase in line 115 - now it will show "Updating..." when we're actually creating the stack, I'll fix it

Reply:
I think it'll be good to support any _action_, and one action we do not support with this implementation is `delete`.

Otherwise we're creating a scenario here that on `sls remove` we will not show any logs. As we unconditionally hide old ones, and show modern ones only on _create_ and _update_.

I would from start in this case also add support for `Deleting CloudFormation stack` operation

Reply:
I did that on purpose, because the PR does not support `sls remove` command at all. I was planning to add it here when we add support for `remove` method. Do you think it's worth supporting `remove` at the moment where all other logs for `remove` are not supported at all?

Reply:
I understand, still I don't think we should create a situation where for some scenario we hide logs without replacement.

So proper way would be either:
- Hide and show logs conditionally (so legacy logs are also replaced with `legacy.*` interface only in case of _create_ or _update_ and in other we rely on old interface)
- Cover all scenarios this logic handles with modern logs.

I think latter is more viable (no weird code changes to make), as I think there's no real reason why  stack update handling that's made in `sls remove` handling will differ from what's happening in `sls deploy`.

Then later when we will focus on `sls remove` we can fine tune, so it shows desired output

Reply:
Added support for `remove` :+1: